nucleus/common/common-util/src/main/java/com/sun/enterprise/universal/process/ProcessUtils.java

```diff
@@ -179,15 +190,15 @@ public static String killJvm(String classname) {
      * @return true if it's running, false if not and null if we don't know. I.e
      * the return value is a true tri-state Boolean.
      */
-    public static Boolean isProcessRunning(int aPid) {
+    public static ProcessState isProcessRunningState(int aPid) {
```

Comment:
Method name sounds strange now that this is actually returning some state enum. Shouldn't it be get...?